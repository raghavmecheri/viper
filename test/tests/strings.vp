string test = "bla";
print(test);
string test2 = "hi";
string test3 = test2;
print(test3);

test = "hello";
print(test);


print("@!#$*(YW(EF*H))");
print("\rwow\twow\nwow\rwow\twow\nwow");
print("11234569097654");