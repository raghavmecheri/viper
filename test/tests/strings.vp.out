bla
hi
hello
@!#$*(YW(EF*H))
\rwow\twow\nwow\rwow\twow\nwow
11234569097654